### DISCUSSION TITLE
ESP8266 SSL example

### DISCUSSION THREAD

**Discussion by miloit** [khoih-prog/AsyncMQTT_Generic#21] [Q&A]
**Is your feature request related to a problem? Please describe.**
Please provide support for SSL support also for ESP8266

**Comment by khoih-prog** [comment]
You can have a look at [**FullyFeaturedSSL_ESP32**](https://github.com/khoih-prog/AsyncMQTT_Generic/tree/main/examples/ESP32/FullyFeaturedSSL_ESP32) example, then adapt / port easily to ESP8266.

Ask for help on Arduino Forum or [**Discussions**](https://github.com/khoih-prog/AsyncMQTT_Generic/discussions) if you still can't do it.

No plan to add ESP8266 SSL example in the near future.
